application-analytics-ui/src/main/resources/Analytics/Code/AnalyticsTranslations.xml

```diff
@@ -149,6 +149,8 @@ analytics.livetable.startDate=Start Date
 analytics.livedata.action.rowEvolution.title=Row Evolution
 
 ## Others
+
+analytics.accessDenied=Please contact the administrator to resolve this issue.
```

Comment:
My comment was only for the second part of the translation, so to have `You do not have access to view this macro. Please contact the administrator to resolve this issue.`